File: 3227-VowelsGameinaString/3227-VowelsGameinaString.js
// Last updated: 3/12/2026, 2:30:46 PM
1/**
2 * @param {string} s
3 * @return {boolean}
4 */
5var doesAliceWin = function(s) {
6    for(let i=0;i<s.length;i++){
7        if(s[i]=="a" || s[i]=="e" || s[i]=="i" || s[i]=="o" || s[i]=="u"){
8            return true;
9        }
10    }
11    return false;
12};

File: 3227-VowelsGameinaString/3227-VowelsGameinaString.cpp
// Last updated: 4/2/2026, 3:40:57 PM
1class Solution {
2public:
3    bool doesAliceWin(string s) {
4        for (int i = 0; i < s.size(); i++) {
5            if (s[i] == 'a' || s[i] == 'e' || s[i] == 'i' || s[i] == 'o' ||
6                s[i] == 'u') {
7                return true;
8            }
9        }
10        return false;
11    }
12};